logstash-core/src/main/java/org/logstash/Event.java

```diff
@@ -264,16 +264,16 @@ public Event clone() throws CloneNotSupportedException {
     }
 
     public String toString() {
-        // TODO: (colin) clean this IOException handling, not sure why we bubble IOException here
-        try {
-            return (getTimestamp().toIso8601() + " " + this.sprintf("%{host} %{message}"));
-        } catch (IOException e) {
-            String host = (String)this.data.get("host");
-            host = (host != null ? host : "%{host}");
+        Object f = this.getField("host");
```

Comment:
it could, both are ok IMO. I decided to reuse it because it is temporary in nature just to set the `host` and `message` variables.